rust/routee-compass-core/src/model/traversal/default/elevation/elevation_traversal_model.rs

```diff
@@ -40,7 +40,7 @@ impl TraversalModel for ElevationTraversalModel {
             ),
             (
                 String::from(fieldname::EDGE_GRADE),
-                InputFeature::Grade(Some(super::ELEVATION_GRADE_UNIT)),
+                InputFeature::Grade(None),
```

Comment:
was this swapped for None by mistake? was `Some(super::ELEVATION_GRADE_UNIT)` which is Decimal